CONTRIBUTING.md

```diff
@@ -1,358 +1,85 @@
-
-1. First occurrence: We'll give you a friendly, but public reminder that the
-   behavior is inappropriate according to our guidelines.
-
-2. Second occurrence: We will send you a private message with a warning that
-   any additional violations will result in removal from the community.
-
-3. Third occurrence: Depending on the violation, we may need to delete or ban
-   your account.
+We currently use [dep](https://github.com/golang/dep) for dependency management.
 
-**Notes:**
+[good-first-issue]: https://github.com/cnabio/cnab-to-oci/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22
+[help-wanted]: https://github.com/cnabio/cnab-to-oci/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22
 
-* Obvious spammers are banned on first occurrence. If we don't do this, we'll
-  have spam all over the place.
+# Cutting a Release
 
-* Violations are forgiven after 6 months of good behavior, and we won't hold a
-  grudge.
+When you are asked to cut a new release, here is the process:
 
-* People who commit minor infractions will get some education, rather than
-  hammering them in the 3 strikes process.
+1. Figure out the correct version number, we follow [semver](semver.org) and
+   have a funny [release naming scheme][release-name]:
+   - Bump the major segment if there are any breaking changes.
+   - Bump the minor segment if there are new features only.
+   - Bump the patch segment if there are bug fixes only.
+   - Bump the build segment (version-prerelease.BUILDTAG+releasename) if you only
+     fixed something in the build, but the final binaries are the same.
+1. Figure out if the release name (version-prerelease.buildtag+RELEASENAME) should
+   change.
 
-* The rules apply equally to everyone in the community, no matter how much
-    you've contributed.
+   - Keep the release name the same if it is just a build tag or patch bump.
+   - It is a new release name for major and minor bumps.
 
-* Extreme violations of a threatening, abusive, destructive or illegal nature
-    will be addressed immediately and are not subject to 3 strikes or forgiveness.
+   If you need a new release name, it must be conversation with the team.
+   [Release naming scheme][release-name] explains the meaning behind the
+   release names.
 
-* Contact abuse@docker.com to report abuse or appeal violations. In the case of
-    appeals, we know that mistakes happen, and we'll work with you to come up with a
-    fair solution if there has been a misunderstanding.
+1. Ensure that the master CI build is passing, then make the tag and push it.
 
-## Coding Style
+   ```
+   git checkout master
+   git pull
+   git tag VERSION -a -m ""
+   git push --tags
+   ```
 
-Unless explicitly stated, we follow all coding guidelines from the Go
-community. While some of these standards may seem arbitrary, they somehow seem
-to result in a solid, consistent codebase.
+1. Generate some release notes and put them into the release on GitHub.
```

Comment:
I believe this repo will use @vdice 's brigade release magic (same as cnab-go), which generates the release notes for you. So this entry can be removed too. 🎩 ✨

Reply:
I'll add an issue related to this then 👍